<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Designer</class>
 <widget class="QMainWindow" name="Designer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Icon maker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QPushButton" name="exportButton">
      <property name="text">
       <string>Export</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icons/export</normaloff>:/icons/export</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="toolsDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="toolsDockLayout">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="textPage">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="textGroup">
           <property name="title">
            <string>Text settings</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="textLabel">
              <property name="text">
               <string>Text</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="Slider" name="pointSizeSlider">
              <property name="minimum">
               <number>6</number>
              </property>
              <property name="maximum">
               <number>256</number>
              </property>
              <property name="value">
               <number>96</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="textEdit">
              <property name="text">
               <string>Aa</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="pointSizeLabel">
              <property name="text">
               <string>Point Size</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="textXLabel">
              <property name="text">
               <string>X axis align</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QPushButton" name="textColourButton">
              <property name="text">
               <string>Text color</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="Slider" name="textXSlider">
              <property name="minimum">
               <number>-64</number>
              </property>
              <property name="maximum">
               <number>64</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="textYLabel">
              <property name="text">
               <string>Y axis align</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="Slider" name="textYSlider">
              <property name="minimum">
               <number>-64</number>
              </property>
              <property name="maximum">
               <number>64</number>
              </property>
              <property name="value">
               <number>-8</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <layout class="QHBoxLayout" name="textFormatLayout">
              <item>
               <widget class="QToolButton" name="boldButton">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>B</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="italicButton">
                <property name="font">
                 <font>
                  <italic>true</italic>
                 </font>
                </property>
                <property name="text">
                 <string>I</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="underlineButton">
                <property name="font">
                 <font>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="text">
                 <string>U</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="6" column="0" colspan="2">
             <widget class="QFontComboBox" name="fontCombo"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="shapePage">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="shapeGroup">
           <property name="title">
            <string>Shape settings</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>Circle</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Square</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="ellipseSizeLayout">
              <item>
               <widget class="QLabel" name="ellipseSizeLabel">
                <property name="text">
                 <string>Scale (%)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Slider" name="ellipseSizeSlider">
                <property name="minimum">
                 <number>10</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>90</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="penSizeLayout">
              <item>
               <widget class="QLabel" name="penSizeLabel">
                <property name="text">
                 <string>Pen size (px)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Slider" name="penSizeSlider">
                <property name="minimum">
                 <number>0</number>
                </property>
                <property name="maximum">
                 <number>64</number>
                </property>
                <property name="value">
                 <number>16</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="colourLayout">
              <item>
               <widget class="QPushButton" name="brushColourButton">
                <property name="text">
                 <string>Brush colour</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="penColourButton">
                <property name="text">
                 <string>Pen colour</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="imagePage">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QGroupBox" name="imageGroup">
           <property name="title">
            <string>Image settings</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="2" column="1">
             <widget class="Slider" name="imageYSlider">
              <property name="minimum">
               <number>-128</number>
              </property>
              <property name="maximum">
               <number>128</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="imageYLabel">
              <property name="text">
               <string>Y axis align</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="imageXLabel">
              <property name="text">
               <string>X axis align</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="imageScaleLabel">
              <property name="text">
               <string>Scale (%)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="Slider" name="imageXSlider">
              <property name="minimum">
               <number>-128</number>
              </property>
              <property name="maximum">
               <number>128</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="Slider" name="imageScaleSlider">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="value">
               <number>90</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="layerDock">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Layers</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="layerDockLayout">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QListView" name="layerView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="addButton">
         <property name="toolTip">
          <string>Add layer</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removeButton">
         <property name="toolTip">
          <string>Remove layer</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="renameButton">
         <property name="toolTip">
          <string>Rename layer</string>
         </property>
         <property name="text">
          <string>Aa</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/edit</normaloff>:/icons/edit</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="upButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Move layer upwards</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/up</normaloff>:/icons/up</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="downButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Move layer downwards</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/down</normaloff>:/icons/down</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Slider</class>
   <extends>QSlider</extends>
   <header>slider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
